generic_tracers/cobalt_glbl.F90

```diff
@@ -0,0 +1,1531 @@
+!> COBALT_glbl module consists of core parameters and vars 
+!! to be used by generic COBALT related modules
+!<----------------------------------------------------------------
+module cobalt_glbl
+  use field_manager_mod, only: fm_string_len, fm_path_name_len      
+  implicit none
+  public  
+
+  !The following variables for using this module
+  ! are overwritten by generic_tracer_nml namelist
+  logical :: do_nh3_diag                             !< logic for setting NH3 diagnostic tracer field 
+
+! Namelist Options
+
+  character(len=10) ::  co2_calc = 'ocmip2'          !< carbonate formalation options. Other option is 'mocsy'
```

Comment:
This could be a separate PR, but now might be a good time to have this default to "mocsy".

Reply:
Based on John's comments, I think this would be safe change to make and could be made now.  

There was an comment about wanting to remove "ocmip" from the name of the old routine (and the routine itself), but perhaps we can address that in a different PR.

Reply:
Change `co2_calc = 'mocsy'`